packages/coding-standard/tests/Fixer/Annotation/DoctrineAnnotationNewlineInNestedAnnotationFixer/Fixture/attribute_overrides.php.inc

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace Symplify\CodingStandard\Tests\Fixer\Annotation\DoctrineAnnotationNewlineInNestedAnnotationFixer\Fixture;
+
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * @ORM\AttributeOverrides(@ORM\AttributeOverride(name="...", column=@ORM\Column(type="...")))
+ */
+class SomeEntity
+{
+}
+
+?>
+-----
+<?php
+
+namespace Symplify\CodingStandard\Tests\Fixer\Annotation\DoctrineAnnotationNewlineInNestedAnnotationFixer\Fixture;
+
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * @ORM\AttributeOverrides(
+ * @ORM\AttributeOverride(name="...", column=
+ * @ORM\Column(type="..."
+ * ))
```

Comment:
This looks correct and test passes :heavy_check_mark: It's mabye fixed on `master`?

I'll merge it to make sure the code does not regress here.

Reply:
Yes, after updating to v9.0.31 it's fixed! 😄  Thanks!

Reply:
Thank you. Good to have it confirmed :wink: 

If you have any troubles in the future, you can send failing fixture without creating issue. It speed the whole process of issue resolving for us, as we see exactly where the build fails.

Thank you for helping us :+1: